internal/validator/helpers.go

```diff
@@ -0,0 +1,53 @@
+package validator
+
+import (
+	"context"
+
+	"github.com/bitcoin-sv/arc/internal/beef"
+)
+
+type HexFormat byte
+
+const (
+	RawHex HexFormat = iota
+	EfHex
+	BeefHex
+)
+
+type TxFinderI interface {
+	GetRawTxs(ctx context.Context, ids []string) ([]RawTx, error)
+}
+
+type RawTx struct {
+	TxID    string
+	Bytes   []byte
+	IsMined bool
+}
+
+type MerkleVerifierI interface {
+	Verify(ctx context.Context, request []beef.MerkleRootVerificationRequest) ([]uint64, error)
+}
```

Comment:
I think these are important definitions of important structs, consts and interfaces. I would move it to `validator.go` with other important definitions, and leave just helper functions `GetHexFormat()` and `ifEf()` here. WDYT?

Reply:
These interfaces are on a different abstraction level; they represent a deeper abstraction. That's why I don't want to mix them with the content from `validator.go`. Ideally, I would put them in the `default` and `beef` packages, but right now I cannot due to cyclic dependencies in the mocks.

If you have a better name for the file than `helper.go`, I will change it, but I'm against moving them to the `validator.go`.